smart-contracts/scripts/multisig/submitTx.js

```diff
@@ -21,7 +21,7 @@ const safeServiceURLs = {
   42161: 'https://safe-transaction-arbitrum.safe.global',
   43114: 'https://safe-transaction-avalanche.safe.global/',
   42220:
-    'https://transaction-service.gnosis-safe-staging.celo-networks-dev.org',
+    'https://safe-transaction-celo.safe.global/',
```

Comment:
also in this PR added the latest endpoint for Celo safe - so we have the latest UI